### ISSUE TITLE
Earnings info disappears / Lower earnings while using MPM / CPU miner closes and disappears from CPU list

### ISSUE THREAD

**Issue by l33twannabe** [aaronsace/MultiPoolMiner#619]
Hello ! 3 issues here that I'm having with MPM.

#
[2017-12-10_21-55-18.txt](https://github.com/aaronsace/MultiPoolMiner/files/1546080/2017-12-10_21-55-18.txt)
1. In the last 2-3 days I've been getting lower earnings with MPM in comparison to the "fastest gpu/cpu miner combo" that MPM shows below. Already tried restprofit.bat and resetbenchmark.bat, but that didn't help.
#2. Since the #1 issue started happening, the earnings info in MPM started to disappear and it's gone for most of the time.
#3. From time to time the CPU miner closes and disappears from CPU miners list. I only use Xmrigcpu and Fireicecpu as these are the most profitable for me. After I restart MPM, both CPU miners appears on the list and are working fine ... until they close and disappears again.

Thanks for your help !

**Comment by Vxrl** [comment]
<span>#</span>2: The profit comparison wasn't displaying for me on one of my computers; removing some code in MultiPoolMiner.ps1 fixed the problem. See https://github.com/aaronsace/MultiPoolMiner/issues/588#issue-280547912 for my fix.

I don't know what's causing <span>#</span>1 and <span>#</span>3, but it looks like you downloaded the latest master instead of [release 2.7.0.2](https://github.com/aaronsace/MultiPoolMiner/releases/tag/v2.7.0.2). Using the master is not recommended. You should be able to copy your "Stats" folder over to the release version so that you don't need to benchmark again.

**Comment by imrecsoka** [comment]
I have had the same issue but didnt want to complain because it might be that i am a silly.
CPU miner just not there when i check on my machine, after saying watching some tv or whatever it is.
Also earnings on XMR-STAK-CPU is low, maybe something CPU intensive in the background eating the power? 
Earnings display in the lower right seems to have been changed as it only comes on once in a while.

**Comment by electrickrain** [comment]
I'm having the same problem as l33twannabe with #3. CPU mining is working fine, and then it's suddenly gone (FireIce). I'm using 2.7.0.2. FirceIce is the best for me, but I found out that when it disappear it switches to something else that gives me 1/5 of what FireIce earns (XMR-STAK is also low, but works fine when I'm using it alone without MPM). I have tried to just use FireIce with MPM, but with no luck, suddenly it's gone.
